08_cell_line_prediction/nn_diagnostics/nbconverted/nn_dropout_range.py

```diff
@@ -0,0 +1,158 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# In[1]:
+
+
+import os
+import glob
+from math import ceil
+
+import numpy as np
+import pandas as pd
+import matplotlib
+import matplotlib.pyplot as plt
+import seaborn as sns
+
+import pancancer_evaluation.config as cfg
+
+get_ipython().run_line_magic('load_ext', 'autoreload')
+get_ipython().run_line_magic('autoreload', '2')
+
+
+# In[2]:
+
+
+results_dir = os.path.join(
+    cfg.repo_root, '08_cell_line_prediction', 'results', 'tcga_ccle_nn_dropout_range'
+)
+
+num_genes = 8000
+plot_gene = 'KRAS'
+
+
+# In[3]:
+
+
+dropout_dfs = {}
+
+for dropout_file in glob.glob(
+    os.path.join(results_dir,
+                 plot_gene,
+                 f'{plot_gene}_signal_mad_s42_n{num_genes}_d*_classify_learning_curves.tsv.gz')
```

Comment:
I'm assuming s42 refers to the seed, but if it's hardcoded, do you ever planning on trying with a different seed?

Reply:
I'll add it as a parameter to the script(s)!